posthog/client.py

```diff
@@ -211,6 +213,9 @@ def capture(
         require("properties", properties, dict)
         require("event", event, string_types)
 
+        if self.super_properties:
```

Comment:
annoyingly in python capture is not the root - we add other "super properties" (like the lib and version) in `enqueue` because other functions like identify, set, etc. etc. directly call enqueue.

So ideal for this handling to happen there as well

Reply:
TIL

Updated in the latest commit